src/core/radio/trel_interface.hpp

```diff
@@ -265,6 +281,7 @@ class Interface : public InstanceLocator
     uint16_t            mUdpPort;
     Packet              mRxPacket;
     PeerTable           mPeerTable;
+    otTrelCounters      mCounters;
```

Comment:
Can remove this? (if we directly use the counter object instance we get from platform layer).

Reply:
Removed.